tests/e2e/playAsGuest.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { test, expect } from "@playwright/test";
+
+test("Play puzzle as guest and lose", async ({ page }) => {
```

Comment:
Is there a play as guest and win test?

Reply:
There is not unfortunately since I do not know how to predict what the word will be for the guest puzzles